tests/ckeditor/ckeditor.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* globals describe, it, expect */
+
+'use strict';
+
+bender.amd.require( 'ckeditor', 'ckeditor-core' );
+
+describe( 'getPluginPath()', function( CKEDITOR, core ) {
```

Comment:
I'm not so convinced with this solution - this means we have to keep in mind the order of required modules and write them down in every `describe`'s callback. I think this solution would work better:
```js
var modules = bender.amd.require( 'ckeditor', 'ckeditor-core' );
```
And then we can use `modules.ckeditor` etc.

Besides, I feel it's too closely coupled with Mocha (which could be avoided thanks to `bender.defer`).